xrpl/asyncio/ledger/main.py

```diff
@@ -74,18 +86,66 @@ async def get_fee(
     if not response.is_successful():
         raise XRPLRequestFailureException(response.result)
 
-    result = response.result["drops"]
-    if fee_type == "open":
-        fee = cast(str, result["open_ledger_fee"])
-    elif fee_type == "minimum":
-        fee = cast(str, result["minimum_fee"])
-    else:
-        raise XRPLException(
-            f'`fee_type` param must be "open" or "minimum". {fee_type} is not a '
-            "valid option."
+    result = response.result
+    drops = result["drops"]
+    if fee_type:
+        if fee_type == "open":
+            fee = cast(str, drops["open_ledger_fee"])
+        elif fee_type == "minimum":
+            fee = cast(str, drops["minimum_fee"])
+        else:
+            raise XRPLException(
+                f'`fee_type` param must be "open" or "minimum". {fee_type} is not a '
+                "valid option."
+            )
+    else:  # use https://gist.github.com/WietseWind/3e9f9339f37a5881978a9661f49b0e52
+        current_queue_size = int(result["current_queue_size"])
+        max_queue_size = int(result["max_queue_size"])
+        queue_pct = current_queue_size / max_queue_size
+        minimum_fee = int(drops["minimum_fee"])
+        median_fee = int(drops["median_fee"])
+        open_ledger_fee = int(drops["open_ledger_fee"])
+
+        fee_low = round(
+            min(
+                max(minimum_fee * 1.5, round(max(median_fee, open_ledger_fee) / 500)),
+                1000,
+            ),
+        )
+        if queue_pct > 0.1:  # if 'current_queue_size' is >10 % of 'max_queue_size'
+            possible_fee_medium = round(
+                (minimum_fee + median_fee + open_ledger_fee) / 3
+            )
+        elif queue_pct == 0:  # if 'current_queue_size' is 0
+            possible_fee_medium = max(
+                10 * minimum_fee, min(minimum_fee, open_ledger_fee)
+            )
```

Comment:
This seems like it would always produce either `10 * minimum_fee` or `open_ledger_fee`. I'm not sure why the `min(minimum_fee, ...)` is required.

Reply:
I will look into it :)

Reply:
You are right, just a little misconception. I will fix that.